index.html

```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
 
         </ul>
     </section>
+
+    <section class="Projects">
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./img/Snapshoot Portfolio.png" alt="snapshot poortfolio" class="snapshot">
+
+            </header>
+            <div class="info-block">
+                <h1 class="project-title">Tonic</h1>
+                <ul class="frame">
+                    <span class="client">
+                        CANOPY</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="role">Back End Dev</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="year">2015</span>
+    
+                </ul>
+                <span class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </span>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            
+                    <button class="see-project">See Project</button>
+        
+            </div>
+        </section>
+
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./Img/Snapshoot Portfolio (1).png" alt="snapshot poortfolio" class="snapshot">
```

Comment:
This image link is not visible. Its link is broken. I would suggest that instead of using an `img` tag here, look into making this a [background image](https://www.w3schools.com/cssref/pr_background-image.php). 
- Decorations are always backgrounded images, not `img` tags)